frontends/ol-components/src/index.ts

```diff
@@ -1,3 +1,4 @@
+"use client"
```

Comment:
These are all client components for now because they use runtime CSS-in-JS (Emotion) which depends on react useContext.

Marking them as use-client allows importing them into server components.